test/proto/benchmarks/control.proto

```diff
@@ -118,32 +67,45 @@ message ParetoParams {
   double alpha = 2;
 }
 
+message ClosedLoopParams {
+}
+
 message LoadParams {
   oneof load {
-    PoissonParams poisson = 1;
-    UniformParams uniform = 2;
-    DeterministicParams determ = 3;
-    ParetoParams pareto = 4;
+    ClosedLoopParams closed_loop = 1;
+    PoissonParams poisson = 2;
+    UniformParams uniform = 3;
+    DeterministicParams determ = 4;
+    ParetoParams pareto = 5;
   };
 }
 
+// presence of SecurityParams implies use of TLS
+message SecurityParams {
+  bool use_test_ca = 1;
```

Comment:
The real prod server issue came up in discussion, in case we ever want to make this happen in the future. To match the interop tests, it was decided to add the option now so that it doesn't later get added as a default false when we'd been assuming it true.

Can we separate the TlsSecurityParams issue into a separate PR?